assets/js/googlesitekit/widgets/register-defaults.js

```diff
@@ -159,4 +168,37 @@ export function registerDefaults( widgetsAPI ) {
 			AREA_DASHBOARD_POPULARITY,
 		],
 	);
+
+	widgetsAPI.registerWidgetArea(
+		AREA_MODULE_SEARCH_CONSOLE_MAIN,
+		{
+			priority: 1,
+			subtitle: ' ',
```

Comment:
The IB mentions using blank/empty strings for the title/subtitle here, why are `' '` strings used throughout instead?

Reply:
Hey @tofumatt 
it's because if we use empty strings, we get the following error message:
`Uncaught Invariant Violation: settings.title is required.`
I don't think we'll need to review the condition which requires the title in case we are using it in some other ways. Also, not sure if the title and subtitle will remain blank later because it looks to me they are placeholders for now.

Reply:
@felixarntz What's the reason behind these titles/subtitles being blank in the issue? Since we explicitly have a check that forbids empty titles/subtitles (and it doesn't seem like a good idea), should we deal with this another way?

I think either we should add:

1. titles/subtitles for these areas
2. a way to hide the title/subtitle but still register them (if the intent here was not to visually show said title/subtitle)

But the current "fake blank" approach feels like a hack and something we're doing to get around rules we explicitly set up, so probably isn't the right approach. 🙂 

Thoughts @felixarntz?

Reply:
@tofumatt @asvinb The titles and subtitles need to be empty because this is to replicate the current module pages ("Search Console" etc.) via the Widgets API. In those pages, there is only the overall page headline (the module title), and then a single widget area for the entire page basically. So this requirement is valid and should be kept as specified in the issue.

However, I agree the way we hack around our own API here is not great. Maybe we can do the following:
* Specify all widget area `title`s as `__( 'Overview', 'google-site-kit' )`.
* Do not specify any `subtitle` for them.
* Make `subtitle` optional in `registerWidgetArea` - that is really something that shouldn't be required.
* Update `WidgetAreaRenderer` so that it does not render the area's header entirely if this widget area is the **only** widget area in its context.